src/siteconfig/tests/test_models.py

```diff
@@ -103,3 +107,35 @@ def test_SiteConfig_cache_expires(self):
         cache_time_expiration = localtime() + timedelta(days=1)
         with freeze_time(cache_time_expiration, tz_offset=0):
             self.assertIsNone(cache.get(SiteConfig.cache_key()))
+
+    def test_get_deck_owner__correct_initialization_default_value(self):
+        """
+            Test to make sure new decks have the expected default deck_owner, as set in settings.py via .env
+        """
```

Comment:
I think I'm confused by the test name "get_deck_owner" because it doesn't match the field name "deck_owner" or the method name "get_default_deck_owner" so I don't even know what you think you are testing here.

I'd like to see the method explicitly tested.  Call the method, compare return value, one test for each of the two possible return values (or in the same method but with a docstring that makes clear what's going on"

Reply:
Leave these tests if they are passing (except change their names to match what they are testing: the deck owner field?)

And create two new test for the method itself.

Reply:
changed name to test_deck_owner__correct_default_value

Also im not exactly sure what to put in the two new test since im only testing one thing and its to check if deck_owner is correctly set to owner user